opm/material/fluidsystems/blackoilpvt/LiveOilPvt.hpp

```diff
@@ -403,6 +403,25 @@ class LiveOilPvt
 
             updateSaturationPressure_(regionIdx);
         }
+
+        // Finalize 2d tables.
+        setLineParallelInterpolation();
+    }
+
+    /*!
+     * \brief Finalize the 2d tables, to be called after all sample points have been inserted.
```

Comment:
it would be nice if the comment explained what the difference to `setParallelInterpolation()` is. Also: this method should not be part of the public API!?